linux_os/guide/system/network/network_sniffer_disabled/ansible/shared.yml

```diff
@@ -11,7 +11,7 @@
 
 - name: "{{{ rule_title }}} - Disable promiscuous mode"
   ansible.builtin.command:
-    cmd: ip link set dev {{ item.split(':')[1] }} multicast off promisc off
+    cmd: ip link set dev {{ (item.split(':')[1] | trim).split('@')[0] }} multicast off promisc off
```

Comment:
Your change looks good to me. However, I have noticed a suspicious thing that exists below on line 16.

```
network_interfaces.stdout_lines is defined and "item.split(':') | length == 3"
```

I think that the double quotes shouldn't be there because they represent a string literal, not a jinja expression. 

I think it should be:

```
network_interfaces.stdout_lines is defined and item.split(':') | length == 3
```


Can you take a look into this?